web/themes/custom/pineapple/sass/components/_hero-image.scss

```diff
@@ -62,30 +62,22 @@
         font-weight: bold;
         font-size: $small-font;
 
-        .field--name-field-r-cooking-time,
-        .field--name-field-r-serves {
-          background-color: transparentize($dark-purple, 0.1);
-          font-weight: 600;
-          color: $white;
+        .inline-field-label {
```

Comment:
It looks like you’re repeating this Inline field sass in your card template, you should create a new file that holds the general set up and similarities of the inline fields and then the hero image and card can alter the colours since thats the only differences

Reply:
Good catch, thank you! Done.